packages/vuetify/src/directives/ripple/index.ts

```diff
@@ -109,14 +105,12 @@ const ripples = {
     animation.classList.add('v-ripple__animation--enter')
     animation.classList.add('v-ripple__animation--visible')
     transform(animation, `translate(${x}, ${y}) scale3d(${scale},${scale},${scale})`)
-    opacity(animation, 0)
     animation.dataset.activated = String(performance.now())
 
     setTimeout(() => {
       animation.classList.remove('v-ripple__animation--enter')
       animation.classList.add('v-ripple__animation--in')
       transform(animation, `translate(${centerX}, ${centerY}) scale3d(1,1,1)`)
-      opacity(animation, 0.25)
```

Comment:
Does this change the appearance of ripples? The stylesheet uses 0.15

Reply:
Yes, it does. it overrides sass values.

Reply:
Should update the sass value to match the current design.

Reply:
Done  👍

Reply:
Any update on this PR ?